packages/amplify-cli/src/__tests__/executeAmplifyCommand.test.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs-extra';
+import { executeAmplifyCommandForPlugin } from '../../lib/execution-manager';
+
+describe('executeAmplifyCommand: ', () => {
+  const mockExecuteAmplifyCommand = jest.fn();
+  const mockContext = jest.fn();
```

Comment:
I don't think chaining this would affect the outcome of your test cases, but for the sake of consistency, should this `mockContext` var be like an empty object instead of a function? It looks like your `executeAmplifyCommandForPlugin()` func accepts a `Context` object as an argument.

Reply:
You are right, I just went for the default jest.fn() and forgot that `context` is an object and not a function.